pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchsvn, pkgconfig, makeDesktopItem, unzip, fpc, lazarus,
+libX11, glib, gtk, gdk_pixbuf, pango, atk, cairo }:
+
+stdenv.mkDerivation rec {
+  name = "transgui-svn-r${revision}";
+  revision = "986";
+
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/transgui/code/trunk/";
+    rev = revision;
+    sha256 = "0z83hvlhllm6p1z4gkcfi1x3akgn2xkssnfhwp74qynb0n5362pi";
+  };
+
+  buildInputs = [ pkgconfig unzip fpc lazarus stdenv.cc libX11
+                  glib gtk gdk_pixbuf pango atk cairo
+                ];
+
+  makeFlags = [
+    "FPC=fpc"
+    "PP=fpc"
+    "INSTALL_PREFIX=$(out)"
+    "LCL_PLATFORM=gtk2"
+  ];
+
+ desktopItem = makeDesktopItem rec {
+    name = "transgui";
+    exec = name;
+    icon = name;
+    comment = meta.description;
+    desktopName = "Transmission Remote GUI";
+    genericName = "BitTorrent Client";
+    categories = "Application;Network;FileTransfer;P2P;GTK;";
+    mimeType = stdenv.lib.concatStringsSep ";" [
+      "application/x-bittorrent" "x-scheme-handler/magnet"
+    ];
+  };
+
+  NIX_LDFLAGS="-L${stdenv.cc.cc.lib}/lib
+    -lX11 -lglib-2.0 -lgtk-x11-2.0 -lgdk-x11-2.0
+    -lgdk_pixbuf-2.0 -lpango-1.0 -latk-1.0 -lcairo -lc";
+
+  preBuild = ''
+    export LCL_PLATFORM=gtk2
```

Comment:
This also seems redundant, given `makeFlags` above. Note also that you could have set this directly via an attribute named `LCL_PLATFORM`.